Makefile

```diff
@@ -0,0 +1,8 @@
+dev-install:
```

Comment:
I expect most people who will install the `reasoner-validator` with make after cloning the repo are doing it for development, (otherwise they would directly install the package from pypi), so maybe we could make `poetry install --all-extras` the default `make install`?

Reply:
done.